### ISSUE TITLE
new individual request: out of scope

### ISSUE THREAD

**Issue by nicolevasilevsky** [information-artifact-ontology/ontology-metadata#77]
For Mondo, we'd like to request a new individual 'out of scope', that is an obsolesence reason.

My ORCID is: https://orcid.org/0000-0001-5208-3432

**Comment by zhengj2007** [comment]
'out of scope' is one of 'obsolescence reason specification' options

From nicolevasilevsky:
>  This is for the Mondo Disease Ontology, where we bring in terms from other sources but sometimes they are considered out of scope for Mondo but they still exist in another source ontology. 

Comments from alanruttenberg:
> It doesn't seem that 'out of scope' would be a reason for obsolescence on
its own unless the class was duplicated in another ontology, in which case
the duplication would be the reason for obsoleting. Otherwise an
out-of-scope term would migrate to the appropriate ontology, no?

More discussions see: https://github.com/information-artifact-ontology/ontology-metadata/pull/75

**Comment by matentzn** [comment]
I think the comment by @nicolevasilevsky is a bit misleading here. The use case is this:

1. You added a term believing it is in scope (`Apple` in a `Salad Ontology`).
2. You later realise `Apple` is not a `Salad`, and want to retire the term
3. There is no fruit ontology. Even if there was, `duplication` is not the right reason for obsoletion, because the true reason is may still be that it is out of scope.

Not sure this is worth a fundamental discussion. I want to obsolete my Apple, please let me do that :)

**Comment by zhengj2007** [comment]
Hi @matentzn and @nicolevasilevsky, Could you please add the definition or example for usage to the term 'out of scope'? So, people can use it correctly. Thanks!

**Comment by zhengj2007** [comment]
OBIB defined 'medication role'. Later we realized the term should be in more general scope ontology. So, the term was submitted and added in the OGMS and deprecated in the OBIB. In OBIB, we set 'obsolescence reason' as 'term imported' and also provided 'term replaced by' annotation with OGMS 'medication role' IRI. 

I am wondering whether the 'out of scope' is more suitable obsolescence reason in this case. @matentzn @alanruttenberg
